Lib/copy.py

```diff
@@ -113,10 +114,7 @@ def _copy_immutable(x):
 for t in (type(None), int, float, bool, complex, str, tuple,
           bytes, frozenset, type, range, slice,
           types.BuiltinFunctionType, type(Ellipsis), type(NotImplemented),
-          types.FunctionType, weakref.ref):
-    d[t] = _copy_immutable
-t = getattr(types, "CodeType", None)
-if t is not None:
```

Comment:
This dance was no longer needed. `types.CodeType` was only missing in `rexec` which was removed in Python 3.